config/initializers/inflections.rb

```diff
@@ -14,3 +14,7 @@
 # ActiveSupport::Inflector.inflections(:en) do |inflect|
 #   inflect.acronym 'RESTful'
 # end
+
+ActiveSupport::Inflector.inflections do |inflect|
+  inflect.irregular 'bonus_reserve', 'bonus_reserves'
+end
```

Comment:
So much fun with this one... Rails was creating routes with a wrong singularization, and was also unable to properly initialize the Game model as it was expecting "bonus_reserf" to be the singularization of "bonus_reserves"... Took me some time to understand where it came from!